gum/backend-darwin/gumprocess-darwin.c

```diff
@@ -6,6 +6,7 @@
  */
 
 #include "gumprocess-priv.h"
+#include "gumprocess-darwin-priv.h"
```

Comment:
This should be moved to the next group, as the first group should only have one include, which should be the main header for this C source file. (The next group contains internal includes, so this include should go there.)